ql/src/test/queries/clientpositive/ptf_register_use_2.q

```diff
@@ -0,0 +1,2 @@
+-- session 2: use function
+select ptf_register_use_func() over();
```

Comment:
before the query, you need add its explain test `explain select ptf_register_use_func() over();`